### PR TITLE
Add atari CLI help.

### PR DESCRIPTION
**Pull request by fchirica**

### CODE REVIEW

File: chia/cmds/data.py [hunk 1]
```diff
@@ -263,10 +272,20 @@ def get_root_history(
...
+@data_cmd.command("add_missing_files", short_help="Manually reconstruct server files from the data layer database")
+@click.option(
+    "-i",
+    "--ids",
+    help="List of stores to reconstruct. If not specified, all stores will be reconstructed",
+    type=str,
+    required=False,
+)
+@click.option(
+    "-o/-n", "--override/--no-override", help="Specify if already existing files need to be overridden by this command"
```
Comment (altendky) [inline chia/cmds/data.py:284]: 'replace' might be a generally better word to consider here in the future?

```suggestion
    "-o/-n", "--override/--no-override", help="Specify if already existing files need to be overwritten by this command"
```

File: chia/cmds/data.py [hunk 2]
```diff
@@ -263,10 +272,20 @@ def get_root_history(
...
 
-@data_cmd.command("add_missing_files", short_help="")
-@click.option("-i", "--ids", help="", type=str, required=False)
-@click.option("-o/-n", "--override/--no-override")
-@click.option("-f", "--foldername", type=str, required=False)
+@data_cmd.command("add_missing_files", short_help="Manually reconstruct server files from the data layer database")
+@click.option(
+    "-i",
+    "--ids",
+    help="List of stores to reconstruct. If not specified, all stores will be reconstructed",
```
Comment (altendky) [inline chia/cmds/data.py:279]: A point for the future to make this not be JSON.

File: chia/cmds/data.py [hunk 3]
```diff
@@ -281,10 +300,10 @@ def add_missing_files(ids: Optional[str], override: bool, foldername: Optional[s
...
 
 
-@data_cmd.command("add_mirror", short_help="")
-@click.option("-i", "--id", help="", type=str, required=True)
-@click.option("-a", "--amount", help="", type=int, required=True)
-@click.option("-u", "--url", "urls", help="", type=str, multiple=True)
+@data_cmd.command("add_mirror", short_help="Publish mirror urls on chain")
+@click.option("-i", "--id", help="Store id", type=str, required=True)
+@click.option("-a", "--amount", help="Amount for this mirror", type=int, required=True)
+@click.option("-u", "--url", "urls", help="List of urls published for the coin", type=str, multiple=True)
```
Comment (altendky) [inline chia/cmds/data.py:306]: Especially given that we have cases where you do actually provide a JSON array (list), I wouldn't use list here.  Maybe this...?

```suggestion
@click.option("-u", "--url", "urls", help="URL to publish on the new coin, multiple accepted and will be published to a single coin.", type=str, multiple=True)
```

File: chia/cmds/data.py [hunk 4]
```diff
@@ -301,8 +320,8 @@ def add_mirror(id: str, amount: int, urls: List[str], fee: Optional[str], data_r
     )
 
 
-@data_cmd.command("delete_mirror", short_help="")
-@click.option("-i", "--id", help="", type=str, required=True)
+@data_cmd.command("delete_mirror", short_help="Delete an owned mirror by its coin id")
+@click.option("-i", "--id", help="Store id", type=str, required=True)
```
Comment (altendky) [inline chia/cmds/data.py:324]: Isn't this the mirror coin id?

```suggestion
@click.option("-i", "--id", help="Coin id", type=str, required=True)
```

**Review by altendky — CHANGES_REQUESTED** [review]
Thanks for filling these in.  It's definitely nice to have entries like this in `--help`.

File: chia/cmds/data.py (hunk 2 above)
Comment (fchirica) [reply chia/cmds/data.py:279]: agreed, using `multiple=True` as everything else would be nice

**Review by altendky — APPROVED** [review]

**Pull request merged**